### DISCUSSION TITLE
Add ignore regions

### DISCUSSION THREAD

**Discussion by kilimi** [open-edge-platform/anomalib#187] [Feature Requests] [answered]
- It is common in real project to have some regions that there is no need to find anomaly, for example, those could be serial number on the part. 
- We can set anomaly map to 0 in regions of interests, but would it make sense to not even include those region in feature extraction and so on (to speed up?) ?

**Comment by alexriedel1** [comment, marked as answer]
Considering the variety of different methods that can be used for anomaly detection, I believe it wouldn't make sense to include these cut out regions in the algorithms.

One easy way for you would be to write a dataloader that takes care of this by zeroing some regions in your training data.

The best way would be to incorporate enough training data, covering all of the variance occuring in your normal data

**Comment by samet-akcay** [comment]
I'm moving this to the `Feature Request` section in the `Discussion` channel. We could continue the discussion there.
